.gitmodules

```diff
@@ -0,0 +1,8 @@
+[submodule "smoke/docs"]
+	path = smoke/docs
+	url = git@github.com:withastro/docs.git
+  branch = main
+[submodule "smoke/astro.build"]
+	path = smoke/astro.build
+	url = git@github.com:withastro/astro.build.git
+	branch = main
```

Comment:
This points `./smoke/docs` to `git@github.com:withastro/docs.git` and `smoke/astro.build` to `git@github.com:withastro/astro.build.git`. They both should reflect the `main` branch.

By default, `git pull` doesn't populate these directories unless you explicitly opt-in to that behavior (which most users won't need to do)

Reply:
I should mention that _users_ won't do this often, but our smoke test action can specifically populate them just for that run. Seems much better than the previous system!